src/components/WalletToolkit/WalletDropdown.tsx

```diff
@@ -72,6 +72,13 @@ const useStyles = makeStyles<any>()((theme, { dark }) => ({
 }));
 
 const WalletDropdown = (props) => {
+  if (!process.env.NEXT_PUBLIC_PRIVY_APP_ID) {
```

Comment:
In React, Hooks must be called in the same order during every render. Therefore, avoid placing if statements at the beginning of a component . Instead, handle the conditional logic in WalletToolkit/index.tsx by checking process.env.NEXT_PUBLIC_PRIVY_APP_ID to determine whether to render the WalletDropdown component.

Reply:
@Holybasil Moved the check to ´WalletDropdown´